Pipfile

```diff
@@ -20,6 +20,7 @@ gunicorn = "==20.0.4"
 authlib = "==0.14.1"
 pyjwt = ">=1.7.1"
 pydantic = ">=1.7.2"
+python-dateutil = "==2.8.1"
```

Comment:
Since this library is out of 0.x, I think you should use a version range here (like `pydantic` does above).